sensor_msgs/CompressedImage img
---
sensor_msgs/CompressedImage img_feats
